ViAn/GUI/qtreeitems.h

```diff
@@ -5,16 +5,17 @@
 #include <QObject>
 #include <QWidget>
 #include <string>
+#include <vector>
+#include "bookmarkview.h"
 #define TEXT_LENGTH 15
-
+typedef int ID;
 enum TYPE {PROJECT, VIDEO};
 
-
 class MyQTreeWidgetItem: public QTreeWidgetItem {
 public:
-    explicit MyQTreeWidgetItem(TYPE type, QString name = "", int id = 0);
```

Comment:
id should not be set to a value per default that could be mistaken for a valid id.